applications/StructuralMechanicsApplication/tests/structural_response_function_test_factory.py

```diff
@@ -149,6 +155,19 @@ def test_execution(self):
         self.assertAlmostEqual(self.gradient[nodeId][1],  0.0017745756664132117, 9)
         self.assertAlmostEqual(self.gradient[nodeId][2], -1.5466215093998671e-07, 9)
 
+@KratosUnittest.skipUnless(has_eigensolvers_application,"Missing required application: EigenSolversApplication")
+class TestAdjointEigenfrequencyResponseFunction(StructuralResponseFunctionTestFactory):
+    file_name = "eigenfrequency_response"
```

Comment:
I am not happy with the name of the class, since eigenfrequency sensitivity analysis includes no adjoint problem.

Reply:
the class name also don't fit to the import in ```test_StructuralMechanicsApplication.py```.

Reply:
fixed it